src/javascript/app/base/header.js

```diff
@@ -315,6 +317,7 @@ const Header = (() => {
                 'tnc',
                 'financial_limit',
                 'risk',
+                'excluded_until',
```

Comment:
I think it should be the first item in the list. Because if a client is self-excluded, doesn't have permission to do any of the above.